src/applications/vaos/new-appointment/components/ProviderSelectPage/ScheduleWithDifferentProvider.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default function ScheduleWithDifferentProvider() {
+  return (
+    <>
+      <h2 className="vads-u-font-size--h3 vads-u-margin-bottom--0 vads-u-margin-top--2">
+        If you want to schedule with a different provider
+      </h2>
+      <h3 className="vads-u-font-size--h4 vads-u-margin-bottom--0 vads-u-margin-top--1">
+        Option 1: Call the facility
+      </h3>
+      <p className="vads-u-margin-y--0">
+        Call and ask to schedule with that provider:{' '}
+        <va-telephone contact="1231231234" /> (
```

Comment:
Do we not have access to the facility phone number at this point in the scheduling flow? Just wondering if we can already use more realistic mock data here instead of a placeholder already.